cogs/audio.py

```diff
@@ -1975,14 +2063,19 @@ def can_instaskip(self, member):
                     dur = "{0}:{1:0>2}".format(m, s)
             else:
                 dur = None
-            msg = ("\n**Title:** {}\n**Author:** {}\n**Uploader:** {}\n"
-                   "**Views:** {}\n**Duration:** {}\n\n<{}>".format(
-                       song.title, song.creator, song.uploader,
-                       song.view_count, dur, song.webpage_url))
-            await self.bot.say(msg.replace("**Author:** None\n", "")
-                                  .replace("**Views:** None\n", "")
-                                  .replace("**Uploader:** None\n", "")
-                                  .replace("**Duration:** None\n", ""))
+
+            msg = ("**Author:** `{}`\n**Uploader:** `{}`\n"
+                    "**Duration:** `{}`\n**Rating: **`{:.2f}`\n**Views:** `{}`".format(
+                    song.creator, song.uploader, str(datetime.timedelta(seconds=song.duration)), song.rating,
+                    song.view_count))
+            msg += self._draw_play(song) + "\n"
```

Comment:
This is missing the server argument. 

Should be:

`msg += self._draw_play(song, server) + "\n"`

Reply:
Thanks, I was wondering what caused your issue as I couldn't reproduce it.

Reply:
Hi, sorry for the late reply. I think I messaged you on Discord about the JSON unpacking, that is all I could find. I don't really use Red, but I can do some testing this weekend if you want.